actix-server/src/lib.rs

```diff
@@ -182,6 +182,10 @@ impl Modify for SecurityAddon {
             models::Org,
             models::Note,
             models::Task,
+            models::Deal,
+            models::Link,
+            models::Email,
+            models::Phone,
```

Comment:
Missing models::Contact

Reply:
lmao I spent so long trying to make sure everything as covered, good catch!